src/stylesheets/blocks/_problem.scss

```diff
@@ -0,0 +1,10 @@
+.problem-section {
+  background-color: color(mint-green);
+}
+
+.screen-reader {
```

Comment:
I'd move this to a separate file in the **utilities** directory. Maybe call it **a11y.css**. Then you can put other boilerplate accessibility stuff in there if you need to.